internal/metastore/kv/rootcoord/kv_catalog.go

```diff
@@ -880,7 +873,7 @@ func (kc *Catalog) ListGrant(ctx context.Context, tenant string, entity *milvusp
 		for i, key := range keys {
 			grantInfos := typeutil.AfterN(key, k+"/", "/")
 			if len(grantInfos) != 2 {
-				log.Warn("invalid grant key", zap.String("key", key))
+				log.Warn("invalid grant key", zap.String("key", key), zap.String("k", k))
```

Comment:
key, k, grantKey is kind of confusing here

Reply:
Updated.